### ISSUE TITLE
About margin parameter order

### ISSUE THREAD

**Issue by darkhandz** [mirego/PinLayout#128]
Hey, thanks for the awesome framework! 
But I'm curious about the paramters order in `margin` function.

```
    public func margin(_ top: CGFloat, _ left: CGFloat, _ bottom: CGFloat, _ right: CGFloat) -> PinLayout

    public func margin(_ top: PinLayout.Percent, _ left: PinLayout.Percent, _ bottom: PinLayout.Percent, _ right: PinLayout.Percent) -> PinLayout
```

Why not use `top right bottom left`?
And the `Readme.md` maybe typo...

```
Set individually top, horizontal margins and bottom margin
margin(_ top: CGFloat, _ right: CGFloat, _ bottom: CGFloat, _ left: CGFloat)
margin(_ top: Percent, _ left: Percent, _ bottom: Percent, _ right: Percent)
```

**Comment by lucdion** [comment]
Hi @darkhandz, happy to hear that you like the framework 🙂
Regarding your questions, I though a little bit about the parameter order for the `margin` method. I know that:
* in CSS the order is top, right, bottom and left (clockwise order)
* on iOS, UIEdgeInsets order is top, left, bottom, right (counterclockwise order)

Which one to choose? Even if PinLayout is greatly inspired by CSS absolute positioning, I've decided to go with the family, i.e. iOS, which is currently the only platform supported by PinLayout. Also because I did't found any good reason de choose clockwise order 🤔

Thanks for the documentation typo, it is fixed.
